infra-as-code/bicep/machinelearning.bicep

```diff
@@ -113,8 +113,8 @@ resource storageFileDataContributorForUserRoleAssignment 'Microsoft.Authorizatio
   properties: {
     roleDefinitionId: storageFileDataContributorRole.id
     principalType: 'User'
-    principalId: yourPrincipalId  // Production readiness change: Users shouldn't be using the Prompt flow developer portal in production, so this role
-                                  // assignment would only be needed in pre-production environments.
+    principalId: yourPrincipalId // Production readiness change: Users shouldn't be using the Prompt flow developer portal in production, so this role
+    // assignment would only be needed in pre-production environments.

```

Comment:
Please align with 116. We align comments vertically when there are multiple in the same indent level in the same scope.